schemas/2016-12-01/Microsoft.RecoveryServices.json

```diff
@@ -0,0 +1,3033 @@
+            }
+          ],
+          "description": "Health Recommended Actions"
+        }
+      },
+      "description": "MAB workload-specific Health Details."
+    },
+    "MabFileFolderProtectedItem": {
+      "type": "object",
+      "allOf": [
+        {
+          "properties": {
+            "friendlyName": {
+              "type": "string",
+              "description": "Friendly name of this backup item."
+            },
+            "computerName": {
+              "type": "string",
+              "description": "Name of the computer associated with this backup item."
+            },
+            "lastBackupStatus": {
+              "type": "string",
+              "description": "Status of last backup operation."
+            },
+            "protectionState": {
+              "type": "string",
+              "description": "Protected, ProtectionStopped, IRPending or ProtectionError"
+            },
+            "isScheduledForDeferredDelete": {
+              "oneOf": [
+                {
+                  "type": "boolean"
+                },
+                {
+                  "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Specifies if the item is scheduled for deferred deletion."
+            },
+            "deferredDeleteSyncTimeInUTC": {
+              "oneOf": [
+                {
+                  "type": "integer"
+                },
+                {
+                  "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Sync time for deferred deletion."
+            },
+            "extendedInfo": {
+              "oneOf": [
+                {
+                  "$ref": "#/definitions/MabFileFolderProtectedItemExtendedInfo"
+                },
+                {
+                  "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Additional information with this backup item."
+            }
+          }
+        }
+      ],
+      "description": "MAB workload-specific backup item."
+    },
+    "MabFileFolderProtectedItemExtendedInfo": {
+      "type": "object",
+      "properties": {
+        "lastRefreshedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time when the agent data synced to service."
+        },
+        "oldestRecoveryPoint": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The oldest backup copy available."
+        },
+        "recoveryPointCount": {
+          "oneOf": [
+            {
+             
```

Comment:
@Tiano2017 Here is the '/definitions/Microsoft.Sql/servers/databases'.
Test tools are unable to recognize these kind of defintions where `/` is embedded in name itself. 

This is the reason tests are also failing.

Reply:
That's an issue, right? I was using VS code validation tools to do this, and it shows errors. In that case we can't publish it, right?

Reply:
I'll say that the validation tools need to be updated if they can't handle these scenario which are generated from perfectly valid swagger spec (from where SDKs have been shipped). 
So this is not an issue in definitions, rather a bug in tools.

Reply:
@Tiano2017 What's the update on this? Are validation tools are going to get updated?

Reply:
@DheerendraRathor We can't update the validation tool. We use the same validation library that VS Code is using. If this is rejected by our tools, it's going to be rejected by VS Code as well. I would recommend you update your swagger, or you'll need to ask VS Code to support this format.